index.html

```diff
@@ -850,13 +847,9 @@ <h2>
       <ol>
         <li>Let |json:PushSubscriptionJSON| be a new {{PushSubscriptionJSON}} dictionary.
         </li>
-        <li>Set |json|["endpoint"] to the result of <a data-lt="get the underlying value">getting
-        the underlying value</a> of the {{PushSubscription/endpoint}} attribute given this
-        {{PushSubscription}} object.
+        <li>Set |json|["endpoint"] to [=getting the `endpoint` attribute=] of [=this=].
```

Comment:
This no longer reads like words arranged to form a sentence.  What about "Set the x to *the output/result of* getting the y attribute of this." ?

Reply:
heh, yeah, it was a linguistic stretch. I'll update to "the result of".